packages/docs/components/tab.md

```diff
@@ -110,32 +104,24 @@ Tabs are best used at the top of the page or situated above the content it’s r
 
 
 <Visual variant="positive" content="no-end" class="is-tab-small-sample">
-  <header>
-    <h2>Missions</h2>
-    <p>There have been 49 missions involving various types of spacecraft.</p>
-  </header>
+  <h2>Missions</h2>
+  <p>There have been 49 missions involving various types of spacecraft.</p>
   <OdsTabs aria-label="Missions by type" :active="tabs.active" :tablist="tabs.tablist" id="example-4"></OdsTabs>
 </Visual>
 
 <Visual variant="negative"  content="no-end" class="is-tab-small-sample">
   <OdsTabs aria-label="Missions by type" :active="tabs.active" :tablist="tabs.tablist" id="example-5">
   <template slot="tab-orbiter">
-    <header>
-      <h2>Missions</h2>
-      <p>There have been 8 missions involving orbiters.</p>
-    </header>
+    <h2>Missions</h2>
+    <p>There have been 8 missions involving orbiters.</p>
```

Comment:
whats going on here with the removal of the `<header>` elements? avoiding styles that were being applied we didn't want or something else?

Reply:
They were causing some unintended rendering issues. I've opened a ticket for us to revisit this - if we want header/context content to be a part of the Tab component, we should make the styles specific to Tab (or create Tab Header). If not, we should ensure the styles don't rely on unrelated markup.